UseCase.md

```diff
@@ -34,6 +34,12 @@ BTCPay checkout is no different to any other payment gateway. Your customer gets
 
 For brick and mortar stores, BTCPay has a [web-based Point of Sale](https://mainnet.demo.btcpayserver.org/apps/87kj5yKay8mB4UUZcJhZH5TqDKMD3CznjwLjiu1oYZXe/pos) (POS) App which can be customized. Similarly to the online store, your customer is presented with an invoice that he can pay on the spot. You can create a watch-only wallet on your phone to be notified of the payments through the POS, without the need of any additional software. There are plans to develop the POS mobile app which would make the process even smoother.
 
+### Freelancers & Bill Pay
+
+Send anyone a request for payment by sharing a [Payment Request](PaymentRequests.md) Url. Customize the content and appearance of the payment request. With or without expiry, users can pay the request at any time. BTCPay automatically updates the BTC exchange rate when the user pays the payment request when it's convenient for them. 
```

Comment:
by sharing a [Payment Request](PaymentRequests.md) **Url** .
Not sure you need the url part at the end, sharing payment request sounds good already?